packages/sshnoports/lib/print_version.dart

```diff
@@ -0,0 +1,8 @@
+import 'dart:io';
+
+import 'package:sshnoports/src/version.dart';
+
+/// Print version number
+void printVersion() {
+  stdout.writeln('Version : $packageVersion');
+}
```

Comment:
This was originally in version.dart, but now that version.dart is generated, this code had to be moved.